### PR TITLE
fix last pending test

### PR DESCRIPTION
**Pull request by noahpryor**

### CODE REVIEW

**Comment by cdale77** [comment]
Thanks for the PR @noahpryor! Yesterday evening I merged some changes to the tests, I think that's why the CI tests are failing. I'll take a look soon, wanted to get back to you quickly. Thanks again!

**Comment by noahpryor** [comment]
Hey, no problem, love the gem - been surprised how few people are building shit around activejob

File: spec/job_status_spec.rb [hunk 1]
```diff
@@ -14,10 +13,9 @@
 
     describe "for a complete job" do
 
-      let!(:job) { TrackableJob.perform_later }
-      sleep(10)
-      #clear_performed_jobs
-      it "should return :complete", pending: true do
+      let!(:job) { TestJob.new('foo').enqueue }
```
Comment (cdale77) [inline spec/job_status_spec.rb]: TestJob isn't defined anywhere, which is why the tests are failing.

**Comment by cdale77** [comment]
@noahpryor if you can fix the TestJob issue, or let me know how you intended it to be, etc, I'd be glad to merge this in. Thanks again!

**Comment by cdale77** [comment]
@noahpryor I'm going to close this. Feel free to re-open. Thanks!

**Comment by noahpryor** [comment]
Fair enough, sorry about that, totally missed your earlier message



—
Noah Pryor 

CTO @ Teachable

713-517-8334

**Pull request closed without merging**